scripts/build_server.sh

```diff
@@ -0,0 +1,10 @@
+echo "Build server files..."
+echo "Building CoffeeScript..."
+./node_modules/.bin/coffee -cb --output build/server server
```

Comment:
As export is scoped in the script only, I'm not sure it's a such nuclear weapon, and it let's commands rely on `$PATH` as usual :)

I prefer prepending to `$PATH`, so local versions installed by _npm_ are looked in first place. It ensures you always use the right version (e.g.: if I've installed coffee-script at system level and never updated it, appending to `$PATH` will produces a build with an outdated version, which may implies some unwanted errors). Using local versions allow us to finely control dev environments, a-la-virtualenv ;)